readme.md

```diff
@@ -19,9 +19,11 @@ const firstChunkStream = require('first-chunk-stream');
 
 // unicorn.txt => unicorn rainbow
 fs.createReadStream('unicorn.txt')
-	.pipe(firstChunkStream({chunkLength: 7}, function (chunk, enc, cb) {
-		this.push(chunk.toUpperCase());
-		cb();
+	.pipe(firstChunkStream({chunkLength: 7}, function (err, chunk, enc, cb) {
```

Comment:
The various use cases i thought of are there:
Silently fail:
https://github.com/sindresorhus/first-chunk-stream/blob/master/test.js#L43
Casting the error:
https://github.com/sindresorhus/first-chunk-stream/blob/master/test.js#L72
Normal behavior when errors happen after the first chunk has been found:
https://github.com/sindresorhus/first-chunk-stream/blob/master/test.js#L107